rosa_tutorials/rosa-mobb-template-tutorial.adoc

```diff
@@ -0,0 +1,68 @@
+:_content-type: ASSEMBLY
+[id=“rosa-mobb-<topic>-tutorial”]
+= Tutorial: <Topic>
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-<topic>-tutorial
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-18
+//---
+//date: '2021-06-10'
+//title: ROSA Prerequisites
+//weight: 1
+//tags: ["AWS", "ROSA", "Quickstarts"]
+//authors:
+//  - <author>
+//---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+Introductory text.
+
+== <Heading 2>
+
+Paragraph text.
+
+image::<example-image>.png[<example image text>]
+
+[NOTE]
+====
+Images need to be added to the images folder to build in the content.
+====
+
+=== <Heading 3>
+
+.Prerequisites <formatting>
+
+Here's the link formatting: 
+
+* link:<redhat.com>[External or hard link]
+* xref:../rosa_architecture/rosa-getting-support.adoc#rosa-getting-support[Sample cross reference]
+
+
+.Procedure <formatting>
+
+. <numbered step formatting>
++
+* <bullet formatting: use a bullet for single step processes or non-numbered process> 
++
+[source,terminal]
+----
+$ <sample-code-block>
+----
++
+[NOTE] 
+====
+<sample note formatting, you can use the following admonitions> 
```

Comment:
```suggestion
<sample note formatting; you can use the following admonitions> 
```
@bmcelvee I know we are not strict here, but adding these suggestions whether they are addressed now or later when we have more time for a deeper review dive.